### PR TITLE
L-01 Change Quorum to Account for Abstain Votes

### PR DESCRIPTION
**Pull request by davidlaprade**
This was recommended by OZ's audit

Fixes https://github.com/ScopeLift/flexible-voting/issues/36

### CODE REVIEW

File: test/GovernorCountingFractional.t.sol [hunk 1]
```diff
@@ -324,7 +324,7 @@ contract GovernorCountingFractionalTest is Test {
     assertEq(abstainVotes, abstainVotesCast);
 
     IGovernor.ProposalState status = IGovernor.ProposalState(uint32(governor.state(_proposalId)));
-    if (forVotes > againstVotes && forVotes >= governor.quorum(block.number)) {
+    if (forVotes > againstVotes && forVotes + abstainVotes >= governor.quorum(block.number)) {
```
Comment (apbendi) [inline test/GovernorCountingFractional.t.sol:327]: Nit: With multi-operator clauses like this I really like extra parens for clarity. Personally I would write it as:

```solidity
if ( (forVotes > againstVotes) && ((forVotes + abstainVotes) >= governor.quorum(block.number)) ) {
```

Or at a minimum:

```solidity
if ( forVotes > againstVotes && (forVotes + abstainVotes) >= governor.quorum(block.number) ) {
```

I know this is a subjective preference. Will leave it up to you to decide and self resolve.

File: test/GovernorCountingFractional.t.sol [hunk 2]
```diff
@@ -665,23 +676,17 @@ contract GovernorCountingFractionalTest is Test {
...
+    bool _wasQuorumReached = _forVotes + _abstainVotes >= _quorum;
+    _quorumTest(_voterAddr, _weight, _voteSplit, _wasQuorumReached);
   }
 
   function _quorumTest(
     address _voterAddr,
     uint256 _weight,
     FractionalVoteSplit memory _voteSplit,
-    uint32 _expectedState
+    bool _expectedQuorumReached
```
Comment (apbendi) [inline test/GovernorCountingFractional.t.sol:689]: Nit: much better name but I'd be even more explicit, something like `_isQuroumReachedExpected`

**Review by apbendi — APPROVED** [review]
Looks good! Nits only.

**Review by mds1 — APPROVED** [review]
LGTM!

File: test/GovernorCountingFractional.t.sol (hunk 1 above)
Comment (davidlaprade) [reply test/GovernorCountingFractional.t.sol]: Sure, I'm good with the second. Fixed in 450b444

**Pull request merged**